src/moin/scripts/maint/modify_item.py

```diff
@@ -40,7 +40,7 @@ def run(self, name, meta_file, data_file, revid):
         if 'charset' in rev.meta['contenttype']:
             # input data will have \r\n line endings, output will have platform dependent line endings
             charset = rev.meta['contenttype'].split('charset=')[1]
-            data = rev.data.read().decode(charset)
+            data = str(rev.data)
```

Comment:
did you want to call `str(rev)`?

Reply:
No that rev is not a ProtectedRevision, so it must be reverted to the old code:

    data = rev.data.read().decode(charset)

Reply:
no, the code implementing `__str__` must be in the Revision, the ProtectedRevision then just forwards to that.